internal/log/log.go

```diff
@@ -111,9 +111,9 @@ var (
 	std loggers.Logger = console.Instance
 )
 
-func PrepareLogging(fs afero.Fs, verbose *bool, loggerSpy io.Writer) {
+func PrepareLogging(fs afero.Fs, verbose bool, loggerSpy io.Writer) {
```

Comment:
@warber do you happen to remember why you originally made this a pointer? Is there something I'm missing?

Reply:
@UnseenWizzard hmm good question, no I think it was just an accident

Reply:
Good to hear! As logging still works, we should be fine without pointers here then :D